backend/setup.py

```diff
@@ -74,6 +74,12 @@
             "inspirehep_records = inspirehep.records:InspireRecords",
             "inspirehep_rt = inspirehep.rt:InspireRtExt",
         ],
+        "invenio_base.api_converters": [
+            "doi = inspirehep.pidstore.converters:DOIConverter",
+            "doipath = inspirehep.pidstore.converters:DOIPathConverter",
+            "arxiv = invenio_records_rest.utils:PIDConverter",
+            "arxivpath = inspirehep.pidstore.converters:ArXivPathConverter",
```

Comment:
Do you need to declare `doi` and `arxiv` here, given you're not using them (only the path version)?